packages/xdl/src/project/publishAsync.ts

```diff
@@ -390,12 +394,22 @@ export async function publishAsync(
     });
   }
 
+  // Create project manifest URL
+  const url =
+    options.releaseChannel && options.releaseChannel !== 'default'
+      ? `${response.url}?release-channel=${options.releaseChannel}`
+      : response.url;
+
+  // Create project page URL
+  const projectPageUrl =
+    options.releaseChannel && options.releaseChannel !== 'default'
+      ? `${response.projectPageUrl}?release-channel=${options.releaseChannel}`
```

Comment:
we probably want to handle the case here `projectPageUrl` is undefined and fall back to `url` - just in case - otherwise we need to be sure to time this release around deploying the update to www and, if something happens and it gets rolled back, then we would have a weird output here

Reply:
> eg: maybe we set projectPageUrl to null there is some unexpected value (like no projectPageUrl in response) and then, in the code that prints this value, we don't display a project page url in that case.

Sounds good- I added this check